app/util/templating.py

```diff
@@ -10,6 +10,20 @@
 
 
 @app.context_processor
+def add_config():
```

Comment:
I'm not sure if this is necessary; there is already an abstraction in place to provide the config to the template during server-side rendering in `api.decorators.context_config()`